<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CapsuleShapeSetter</class>
 <widget class="QWidget" name="CapsuleShapeSetter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>56</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Vertex1:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Vertex2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Radius:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Vec3Setter" name="widgetVertex1" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="Vec3Setter" name="widgetVertex2" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="spinBoxRadius">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="maximum">
      <double>999999999.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Vec3Setter</class>
   <extends>QWidget</extends>
   <header>vec3setter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
